python/katana/numba_support/closure.py

```diff
@@ -101,6 +105,12 @@ def _build_wrapper(func, load_struct, return_type, bound_args, unbound_args):
         """
         The arguments are unpacked from the jitclass pointer passed as an int64.
         """
+        # The following parameters are used in exec below, but most linters
+        # will not notice their use there.
+        _ = func
+        _ = return_type
+        _ = load_struct
```

Comment:
I would prefer to explicitly pass these as `locals = dict(func=func, ...)` to `exec`. Using the implicit `locals()` is kind of unclear anyway.